programmers/string-handling-basic.js

```diff
@@ -0,0 +1,20 @@
+function solution(s){
+    const linerMeasure = s.length;
+    if(linerMeasure === 4 | linerMeasure === 6){
+        for (let i = 0 ; i<linerMeasure; i++){
+            let store = 0;
+            for (let j = 0 ;  j<10 ; j++){
+                if (s[i] == j){
+                    store = 1;
+                }
+            }
+            if (store === 0){
+                return console.log("false");
```

Comment:
console.log를 왜 써주시지는지 모르겟어요 ㅠ
```suggestion
                return false;
```